django/contrib/admin/checks.py

```diff
@@ -18,9 +18,13 @@
 
 
 def check_admin_app(**kwargs):
-    from django.contrib.admin.sites import system_check_errors
+    from django.contrib.admin.sites import all_sites
```

Comment:
That sounds reasonable but does leave the question about `AdminSite`s that mixes models for apps that aren't currently installed... I guess this might not happen in practice, at least in Django's test suite which is the main app-registry-fiddler anyway.